src/borg/blake2/libselect.h

```diff
@@ -0,0 +1,5 @@
+#ifdef BORG_USE_LIBB2
+#include <blake2.h>
+#else
+#include "blake2b-ref.c"
+#endif
```

Comment:
hmm, as this is rather part of borg than of blake2 ref implementation, move it to outside of this dir, so maintainers can just kill the whole blake2/ directory to remove it from the package if they use libb2?